### ISSUE TITLE
problem installing with homebrew

### ISSUE THREAD

**Issue by letsop** [maelvls/homebrew-amc#49]
Hello, I tried installing with homebrew and it fails.

(base) Cervin:~ postel$ brew install auto-multiple-choice --build-from-source
Updating Homebrew...
==> Installing auto-multiple-choice from maelvls/amc
==> Installing dependencies for maelvls/amc/auto-multiple-choice: make
==> Installing maelvls/amc/auto-multiple-choice dependency: make
==> Downloading https://homebrew.bintray.com/bottles/make-4.3.high_sierra.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/e7/e7f49884dbeaae3e7ccc9dc3a58fec587c2a594f50314dbc33cde44c6f807f53?__gda__=exp=1587477185~hmac=f13e41d5ed1c50df3f07016dd2a58ea6c30ab
######################################################################## 100.0%
==> Pouring make-4.3.high_sierra.bottle.tar.gz
==> Caveats
GNU "make" has been installed as "gmake".
If you need to use it as "make", you can add a "gnubin" directory
to your PATH from your bashrc like:

    PATH="/usr/local/opt/make/libexec/gnubin:$PATH"
==> Summary
🍺  /usr/local/Cellar/make/4.3: 15 files, 1MB
==> Installing maelvls/amc/auto-multiple-choice
==> Downloading https://gitlab.com/jojo_boulix/auto-multiple-choice/uploads/ae5e224c2490bfcdec676a32b1b476f6/auto-multiple-choice_1.4.0_dist.tar.gz
######################################################################## 100.0%
==> make AMCCONF=brew PREFIX=/usr/local/Cellar/auto-multiple-choice/1.4.0_8 LIBS_PREFIX=/usr/local
==> make install_doc AMCCONF=brew PREFIX=/usr/local/Cellar/auto-multiple-choice/1.4.0_8 LIBS_PREFIX=/usr/local
==> make install_nodoc AMCCONF=brew PREFIX=/usr/local/Cellar/auto-multiple-choice/1.4.0_8 LIBS_PREFIX=/usr/local
==> Downloading https://cpan.metacpan.org/authors/id/S/SB/SBECK/Locale-Codes-3.58.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/J/JO/JOSEPHW/XML-Writer-0.625.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/O/OA/OALDERS/Net-HTTP-6.18.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GA/GAAS/HTTP-Negotiate-6.01.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GA/GAAS/WWW-RobotRules-6.02.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/O/OA/OALDERS/HTTP-Cookies-6.04.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/P/PE/PETDANCE/HTML-Tagset-3.20.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GA/GAAS/HTTP-Daemon-6.01.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GA/GAAS/File-Listing-6.04.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GA/GAAS/HTTP-Date-6.02.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/H/HA/HAARG/Test-Needs-0.002005.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/E/ET/ETHER/URI-1.74.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/C/CJ/CJM/IO-HTML-1.001.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GA/GAAS/Encode-Locale-1.05.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GA/GAAS/LWP-MediaTypes-6.02.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/O/OA/OALDERS/HTTP-Message-6.18.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/M/MA/MALLEN/Test-RequiresInternet-0.05.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/E/ET/ETHER/libwww-perl-6.35.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/T/TO/TODDR/XML-Parser-2.44.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-Base-1.09.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/P/PE/PERIGRIN/XML-NamespaceSupport-1.12.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-SAX-1.00.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/B/BJ/BJOERN/XML-SAX-Expat-0.51.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GR/GRANTM/XML-Simple-2.25.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/I/IS/ISHIGAKI/Text-CSV-1.97.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/J/JC/JCRISTY/PerlMagick-6.89-1.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none INC=-I/usr/local/opt/imagemagick@6/include/ImageMagi
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/M/MI/MIROD/XML-Twig-3.52.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/R/RG/RGARCIA/Sub-Identify-0.14.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/C/CH/CHROMATIC/SUPER-1.20141117.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/L/LE/LEONT/Module-Build-0.4224.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/G/GF/GFRANKS/Test-MockModule-v0.170.0.tar.gz
######################################################################## 100.0%
==> perl Build.PL --install_base /usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec
==> ./Build
==> ./Build install
==> Downloading https://cpan.metacpan.org/authors/id/P/PH/PHRED/Archive-Zip-1.63.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/J/JM/JMGDOC/OpenOffice-OODoc-2.125.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/B/BI/BINGOS/Module-Load-Conditional-0.68.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/P/PV/PVANDRY/gettext-1.07.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/X/XA/XAOC/ExtUtils-Depends-0.405.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/X/XA/XAOC/ExtUtils-PkgConfig-1.16.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/X/XA/XAOC/Glib-1.327.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/X/XA/XAOC/Cairo-1.106.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
==> make
==> make install
==> Downloading https://cpan.metacpan.org/authors/id/X/XA/XAOC/Cairo-GObject-1.004.tar.gz
######################################################################## 100.0%
==> perl Makefile.PL INSTALL_BASE=/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec INSTALLMAN1DIR=none INSTALLMAN3DIR=none
Last 15 lines from /Volumes/USER/postel/Library/Logs/Homebrew/auto-multiple-choice/121.perl:
INSTALLMAN1DIR=none
INSTALLMAN3DIR=none

Can't load '/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec/lib/perl5/darwin-thread-multi-2level/auto/Cairo/Cairo.bundle' for module Cairo: dlopen(/usr/local/Cellar/auto-multiple-choice/1.4.0_8/libexec/lib/perl5/darwin-thread-multi-2level/auto/Cairo/Cairo.bundle, 1): Library not loaded: /usr/local/lib/libpng16.16.dylib
  Referenced from: /usr/local/lib/libfreetype.6.dylib
  Reason: image not found at /usr/local/Cellar/perl/5.30.2_1/lib/perl5/5.30.2/darwin-thread-multi-2level/DynaLoader.pm line 197.
 at (eval 20) line 1.
Compilation failed in require at (eval 20) line 1.
BEGIN failed--compilation aborted at (eval 20) line 1.

Checking if your kit is complete...
Looks good
Warning: NAME must be a package name
Warning: Guessing NAME [Cairo-GObject] from current directory name.
Can't locate object method "postamble_clean" via package "Glib::MakeHelper" (perhaps you forgot to load "Glib::MakeHelper"?) at Makefile.PL line 106.

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
  https://github.com/maelvls/homebrew-amc/issues

(base) Cervin:~ postel$ 
(base) Cervin:~ postel$ 
(base) Cervin:~ postel$ brew linkage auto-multiple-choice
Error: No such keg: /usr/local/Cellar/auto-multiple-choice

**Comment by maelvls** [comment]
Sorry for that, the nightly build had been failing for 5 days now (opencv
new minor version) que I didn’t find time to update the formula

I’ll try to do that ASAP 👍

**Comment by maelvls** [comment]
I am having a hard time maintaining the formula; upstream Homebrew has updated their "ruby API" and it broke pretty much everything in my CI 😞

It will take a will to figure out what went wrong, so I won't be able to push new bottles right away... Sorry for that!

**Comment by maelvls** [comment]
Hi!

I [disabled the bottles](https://github.com/maelvls/homebrew-amc#july-3-2020-%EF%B8%8F-i-dont-have-enough-time-to-maintain-bottles) since I didn't have enough time to fix the CI that was building them. It will take slightly longer to install (8 minutes instead of 30 seconds) but it will work 100% of the time!!

Could you try to run

```sh
brew update
brew reinstall maelvls/amc/auto-multiple-choice
```

It should now install :)

**Comment by maelvls** [comment]
@letsop I tried to fix a couple of bugs (#55) this morning, could you try reinstalling again?

**Comment by maelvls** [comment]
Closing for now. Feel free to re-open if this is still an issue. 👍
